### PR TITLE
MM-1426 - Updating UI

### PR DESCRIPTION
**Pull request by asaadmahmoodspin**
Hey guys have a look.

### CODE REVIEW

**Comment by asaadmahmoodspin** [comment]
Added the highlight stuff too along with the other UI improvements.

File: web/react/components/post.jsx [hunk 1]
```diff
@@ -64,13 +65,18 @@ module.exports = React.createClass({
         }
 
         var postType = "";
-        if(type != "Post"){
+        if (type != "Post"){
             postType = "post--comment";
         }
 
+        var currentUser = "";
```
Comment (coreyhulen) [inline web/react/components/post.jsx]: maybe change var name to currentUserCss so it's clear.

**Comment by coreyhulen** [comment]
I think there are a few more places with currentUser

File: web/react/components/post_right.jsx [hunk 2]
```diff
@@ -68,9 +68,14 @@ RootPost = React.createClass({
...
+        var type = "Post";
         if (this.props.post.root_id.length > 0) {
-            type = "Comment"
+            type = "Comment";
+        }
+
+        var currentUser = "";
+        if (UserStore.getCurrentId() === this.props.post.user_id) {
+            currentUser = "current--user";
         }
```
Comment (rgarmsen2295) [inline web/react/components/post_right.jsx]: Two here

**Pull request merged**